src/components/NavBar/NavBarProducts/NavBarProducts.jsx

```diff
@@ -52,10 +53,12 @@ const NewLabel = styled.span`
   color: ${white};
   font-family: ${fontFamily};
   font-weight: ${fontWeightMedium};
+  font-size: ${fontSizeSmall};
   background: ${green};
-  border-radius: 4px;
+  border-radius: 12px;
   padding: 2px 8px;
   margin-left: 8px;
+  margin-top: 2px;
```

Comment:
This is needed in order to push the label down a little bit to make it visually aligned.